docs/outgoing-webhooks.md

```diff
@@ -0,0 +1,113 @@
+# Outgoing Webhooks
+
+Outgoing webhooks are a type of integration which post data to third party URLs through a HTTP POST request.
+To create an outgoing webhook, go to the `Add a new bot` tab in bot settings, and select the following options:
+* **Bot type**: outgoing webhook
+* **Full name, Username**: as desired, the username will be taken as name of outgoing webhook
+* **Base URL**: desired URL to which data is to be posted
+* **Interface**: generic for simple outgoing webhooks, else desired interface as per use
+
+
+![Add new bot form](images/add_new_bot_form.png)
+
+
+## Editing the details:
+**Base URL** and **Interface** for an outgoing webhook can be edited by users, through the `Edit bot` form.
+To open the `Edit bot` form, go to the `Active bots` tab, and for the outgoing webhook bot to be edited,
+click on the pencil shaped icon below the bot's name.
+
+
+![Edit bot form](images/edit_bot_form.png)
+
+
+## Triggering
+There are currently two methods to trigger an outgoing webhook:
+1.  **@-mentioning the bot which is the owner of outgoing webhook**
```

Comment:
Use the imperative tense for the bullets, like they're instructions.  So say "@-mention the bot...".

Also use "that" instead of "which" here.